src/Content/Conversation.php

```diff
@@ -965,10 +963,21 @@ private function addChildren(array $parents, $block_authors, $order, $uid)
 		$condition = DBA::mergeConditions($condition,
 			["`uid` IN (0, ?) AND (`vid` != ? OR `vid` IS NULL)", $uid, Verb::getID(Activity::FOLLOW)]);
 
+		$params = ['order' => ['uri-id' => false, 'uid' => true]];
+		$thread_parents = Post::select(['uri-id', 'causer-id'], $condition, $params);
```

Comment:
Since `$params` is overwritten on line 976, I'd rather we inline this one or rename the variable.
```suggestion
		$thread_parents = Post::select(['uri-id', 'causer-id'], $condition, ['order' => ['uri-id' => false, 'uid' => true]]);
```
```suggestion
		$paramsParents = ['order' => ['uri-id' => false, 'uid' => true]];
		$thread_parents = Post::select(['uri-id', 'causer-id'], $condition, $paramsParents);
```

Reply:
This can be overwritten, this is not a problem.

Reply:
It isn't a technical issue, but it is a code readability issue.